src/java.base/share/classes/sun/security/pkcs12/PKCS12KeyStore.java

```diff
@@ -1958,10 +1959,21 @@ public synchronized void engineLoad(InputStream stream, char[] password)
         certPbeIterationCount = -1;
         macAlgorithm = null;
         macIterationCount = -1;
+        String storeName = "";
```

Comment:
I think it is cleaner to not initialize this (leave it as null) and check for null inside debug statements. Same comment in `PKCS12KeyStore.java`.